services/galley/src/Galley/Run.hs

```diff
@@ -61,23 +60,24 @@ import qualified Network.Wai.Middleware.Gunzip as GZip
 import qualified Network.Wai.Middleware.Gzip as GZip
 import Network.Wai.Utilities.Server
 import Servant hiding (route)
-import Servant.Client
-  ( BaseUrl (BaseUrl),
-    ClientEnv (ClientEnv),
-    Scheme (Http),
-    defaultMakeClientRequest,
-  )
 import qualified System.Logger as Log
 import Util.Options
 import Wire.API.FederationUpdate
 import Wire.API.Routes.API
 import Wire.API.Routes.FederationDomainConfig
 import qualified Wire.API.Routes.Public.Galley as GalleyAPI
 import Wire.API.Routes.Version.Wai
+import System.Logger.Extended (mkLogger)
 
 run :: Opts -> IO ()
 run opts = lowerCodensity $ do
-  (app, env) <- mkApp opts
+  (ioref, _) <- lift $ do
+    -- Duplicating the logger from App.createEnv so that we don't have to deal
+    -- with recursive monadic actions to get the logger before we have the initial
+    -- IORef of values from brig. It's just easier this way.
+    l <- mkLogger (opts ^. optLogLevel) (opts ^. optLogNetStrings) (opts ^. optLogFormat)
```

Comment:
is this ok?  i'm worried about output from two loggers getting entangled somehow, even though i couldn't explain how.

can't we create the logger first thing in `run`, and then pass it to both `createEnv` and `updateFedDomains`?

Reply:
Done and done.